File: modulo1/segmento3/apunte3/ejercicio3/ejercicio3.java
import java.util.Scanner;

public class ejercicio3{
	public static void main(String args[]){
			Scanner teclado = new Scanner(System.in);
				
			String opcion;

			System.out.println("Ingrese un caracter cualquiera : ");
				String caracter = teclado.nextLine();

			if(caracter.toUpperCase().equals("A") || caracter.toUpperCase().equals("E") || caracter.toUpperCase().equals("I") || caracter.toUpperCase().equals("O") || caracter.toUpperCase().equals("U")){
				opcion = "vocal";
			}else if(isNumeric(caracter)){
				opcion="es numero";
			}
			else{
				opcion="consonante";
			}
			System.out.println(caracter.toString() + " es " + opcion );
			}	
	private static boolean isNumeric(String cadena){
	        try {
				Integer.parseInt(cadena);
			return true;
			} catch (NumberFormatException nfe){
			return false;
			}
		}
}

File: modulo1/segmento3/apunte3/ejercicio3/ejercicio3.py
caracter = input("Ingrese un caracter para saber si es una (vocal, consonante, numero): ")

if(caracter == "1" or "2" or "3" or "4" or "5" or "6" or "7" or "8" or "9"):
	print("Ud ingreso un numero")

elif(caracter == "a" or "e" or "i" or "o" or "u"):
	print("Ud ingreso un vocal")

elif(caracter == "b" or "c" or "d" or "f" or "g" or "h" or "j" or "k" or "l" or "m" or "n" or "p" or "q" or "r" or "s" or "t" or "v" or "w" or "x" or "y" or "z"):
	print("Ud selecciono una consonante")

else:
	print("Ud selecciono un simbolo")
